src/ansiblelint/rules/YamllintRule.py

```diff
@@ -28,6 +28,9 @@
 YAMLLINT_CONFIG = """
 extends: default
 rules:
+  comments:
+    # https://github.com/prettier/prettier/issues/6780
+    min-spaces-from-content: 1
```

Comment:
Why not 2, like in Python?

Reply:
Because this conflicts with other yaml linters that don't allow specifying multiple spaces before the comment. Prettier is one example (see prettier/prettier#6780 as in the comment)

Another example is the `fmt` mode in #1805 (aside: I'm working on splitting that behemoth into smaller PRs) which uses `ruamel.yaml` to edit files to fix issues identified by ansible-lint's rules. `ruamel.yaml` does not have a way for me to specify how many spaces should be inserted before comments in any automated fashion. I would have to manually iterate through the document to check every comment and adjust the columns... Oh pain! (if that would even work) I really hope we don't have to do that.